.changeset/gorgeous-carrots-hammer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@remirror/styles': patch
+---
+
+Update `caniuse` and remove some unneeded CSS prefixes.
```

Comment:
Thank you, next time I will be more descriptive.

Reply:
No worries